### PR TITLE
Update Node versions

### LINKED ISSUE #4846: Update Supported Node Versions for Travis CI
This didn't fit the template quite right, so I'll wing it:

The current _.travis.yml_ file should be updated to [reflect current NodeJS versions](https://github.com/nodejs/Release#release-schedule). The only versions should be **6, 8, 10**.

### PR DESCRIPTION
**Pull request by simonua**
Fixes #4846

### CODE REVIEW

**Comment by simonua** [comment]
@qiyigg, can this be merged, please?

File: package-lock.json [hunk 1]
```diff
@@ -2775,9 +2775,9 @@
       }
     },
     "natives": {
-      "version": "1.1.1",
```
[the remark this answers is not in the document]
Comment (qiyigg) [reply package-lock.json]: oh, I see. It is a gulp issue

**Comment by qiyigg** [comment]
Please take a look at the commit format:
https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#commit-messages

This time I will change it for you.

**Comment by simonua** [comment]
@qiyigg, thank you for updating the commit and the merge. I'll follow this more closely next time.

**Comment by qiyigg** [comment]
No problem, thanks!

**Pull request merged**